src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/LogsController.php

```diff
@@ -88,35 +90,17 @@ public function indexAction(Request $request)
     }
 
     /**
+     * @AdminSecurity("is_granted(['read','update', 'create','delete'], request.get('_legacy_controller')~'_')", message="You do not have permission to update this.", route="admin_logs")
```

Comment:
Maybe use the same syntax as https://github.com/PrestaShop/PrestaShop/pull/9028/files#diff-b220b6c2fe8d243c81424348b32820daR49 (use space after `,` or not :smile: )